### PR TITLE
[security-external-integrations packages] Update to ECS 8.3

### PR DESCRIPTION
**Pull request by andrewkroh**
## What does this PR do?

This updates the ECS version used in all non-deprecated packages owned by elastic/security-external-integrations.

These packages required fixes in order to comply with the `pattern` added to ECS to validate MAC addresses.

- cef - https://github.com/elastic/integrations/pull/3566
- crowdstrike - https://github.com/elastic/integrations/pull/3302
- cylance.protect - https://github.com/elastic/integrations/pull/3368
- fortinet.fortimanager - https://github.com/elastic/integrations/pull/3401
- iptables.log - https://github.com/elastic/integrations/pull/3358
- microsoft_dhcp - https://github.com/elastic/integrations/pull/3300
- pfsense - https://github.com/elastic/integrations/pull/3303
- snort - https://github.com/elastic/integrations/pull/3301
- sonicwall.firewall - https://github.com/elastic/integrations/pull/3360
- sophos.utm - https://github.com/elastic/integrations/pull/3370

NOTE: The following packages were not updated for 8.2.0. I didn't catch anything in 8.1 or 8.2 that needed changed.

 - auth0 - 1.12.0
 - carbon_black_cloud - 8.0.0
 - cisco_ise - 8.0.0
 - cisco_meraki - 8.0.0
 - hid_bravura_monitor - 1.12.0
 - modsecurity - 1.12.0
 - mysql_enterprise - 8.0.0
 - netskope - 8.0.0
 - oracle - 8.0.0
 - symantec_endpoint - 1.12.0
 - ti_recordedfuture - 8.0

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to ~test~ recreate this PR locally

The changeset was fully produced by this command:

```
go run github.com/andrewkroh/go-examples/ecs-update@6efa1ecb3871 \
  --ecs-version=8.3.0 \
  -ecs-git-ref=v8.3.0 \
  --pr=3353 \
  --owner=elastic/security-external-integrations \
  packages/*
```

## Related issues

- Relates #3200

### CODE REVIEW

**Review by DamiaPoquet — COMMENTED** [review]
This change conflicts with https://github.com/elastic/integrations/pull/3347

**Review by r00tu53r — APPROVED** [review]
LGTM

**Comment by ishleenk17** [comment]
@andrewkroh : Which packages are to be reviewed from service integrations side. 
The ones I found are oracle and mysql_enterprise ?

Also, a general query: It would be a good practice to update the ecs versions once the releases are done ?
Do we expect this upgrade to be a breaking change in any scenario ?

**Comment by andrewkroh** [comment]
> Also, a general query: It would be a good practice to update the ecs versions once the releases are done ?

Currently this is pointing to the elastic/ecs 8.3 branch. Once ECS tags its release for v8.3.0 I will re-run the script use the tag instead of the ECS branch. This will prevent changes in the ECS branch from affecting package builds. Because we want to identify the required changes during the 8.3 development cycle, we try to do this work way ahead of the official release so that the work can be pushed to the EPR very close the stack release.

>  Do we expect this upgrade to be a breaking change in any scenario ?

No.

> Which packages are to be reviewed from service integrations side.

This PR updates only packages owned by elastic/security-external-integrations. Of those, the only one where elastic/obs-service-integrations is listed as a shared Github codeowner is `microsoft_sqlserver`. You're welcome to review anything. The changes are generated by script (as mentioned in the description) so an issue in one package is probably an issue in the others.

File: packages/hid_bravura_monitor/data_stream/winlog/sample_event.json [hunk 1]
```diff
@@ -85,6 +85,6 @@
         "type": "filebeat"
     },
     "ecs": {
-        "version": "1.12.0"
+        "version": "8.3.0"
```
Comment (efd6) [inline packages/hid_bravura_monitor/data_stream/winlog/sample_event.json:88]: This file may want to be regenerated.

File: packages/symantec_endpoint/data_stream/log/sample_event.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
     },
     "@timestamp": "2021-11-16T12:14:15.000Z",
     "ecs": {
-        "version": "1.12.0"
+        "version": "8.3.0"
```
Comment (efd6) [inline packages/symantec_endpoint/data_stream/log/sample_event.json:50]: This file may want to be regenerated.

File: packages/ti_recordedfuture/data_stream/threat/sample_event.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
         "type": "logs"
     },
     "ecs": {
-        "version": "8.0"
+        "version": "8.3.0"
```
Comment (efd6) [inline packages/ti_recordedfuture/data_stream/threat/sample_event.json:16]: This file may want to be regenerated.

**Review by efd6 — COMMENTED** [review]
LGTM

The three packages that have not been updated since 1.12 have sample events that do not reflect the current state to a minor degree, either the order of the fields or the implied type of numeric fields. Running all 97 packages would be onerous, but it could be parameterised into the code generator.

**Pull request merged**